MODULE.bazel

```diff
@@ -13,14 +13,15 @@ bazel_dep(name = "bazel_skylib", version = "1.4.2")
 bazel_dep(name = "rules_python", version = "0.29.0")
 bazel_dep(name = "platforms", version = "0.0.7")
 
-archive_override(
-    module_name = "rules_python",
-    urls = "https://github.com/bazelbuild/rules_python/archive/52381415be9d3618130f02a821aef50de1e3af09.tar.gz",
-    integrity = "sha256-pYfEFNWqygSEElDYgJsuIeDYn9oll/rZB0GcR+6rirA=",
-    strip_prefix = "rules_python-52381415be9d3618130f02a821aef50de1e3af09",
-)
+tools = use_extension("//py:extensions.bzl", "py_tools")
+tools.rules_py_tools()
+use_repo(tools, "rules_py_tools")
 
 register_toolchains(
+    "@rules_py_tools//:all",
```

Comment:
are concerned with the order of toolchains being registereted?

Reply:
The ones in this package are guaranteed to be mutually exclusive with each other, so the order shouldn't matter here.